internal/storage/types.go

```diff
@@ -30,6 +30,11 @@ type AddressStats struct {
 	mutex   sync.RWMutex `rlp:"-"`
 }
 
+type Account struct {
+	models.Account
+	mutex sync.RWMutex `rlp:"-"`
+}
+
```

Comment:
I think its better to have something like `Balance *big.Int` here to not parse it on every modification. And convert to a model with string values just before you will save it to db

Reply:
Right, I'll change this